tools/shared.py

```diff
@@ -2396,6 +2396,17 @@ def get_binaryen_bin():
     Building.get_binaryen()
     return os.path.join(Settings.BINARYEN_ROOT, 'bin')
 
+  @staticmethod
+  def get_binaryen_lib():
+    Building.get_binaryen()
+    paths = (os.path.join(Settings.BINARYEN_ROOT, 'bin'),
+             os.path.join(Settings.BINARYEN_ROOT, 'share', 'binaryen'))
```

Comment:
i don't follow, what would the `share/binaryen` use case be?

i was expecting to see something like a check if `bin/` and `lib/` exist, or if we are in the `bin` dir already and need to go one up, something like that, but I guess I misunderstood?